components/docs-chef-io/content/automate/ha_automate_to_automate_ha.md

```diff
@@ -18,52 +16,193 @@ gh_repo = "automate"
 
 {{< warning >}}
 
-- Standalone Chef Automate or Chef Automate with embedded Chef Infra Server can migrate to Automate HA, with  
-minimum version of Chef Automate: [20201230192246](https://docs.chef.io/release_notes_automate/#20201230192246)
-
-- Chef Automate user running Chef Infra Server in external mode should not migrate to Automate HA.
+- The Minimum version of Automate to perform migration is 4.x.
+- Customers in 2022xxxx or 3.x version needs to update to 4.x version.
+- Automate HA with AWS managed needs to use OpenSearch version <1.3.2
+- The OpenSearch version in Automate should be the same version as Automate HA.
 
 {{< /warning >}}
 
 ## Upgrade with FileSystem Backup Locally
 
-1. Create Backup of Chef Automate Standalone using the following command:
+Follow the below steps for Automate HA when you are upgrading with On-Premises and AWS deployment (but not for AWS managed services).
+
+1. Create a Backup of Chef Automate Standalone using the following command:
+
+    Run the below command to create the backup in the `/var/opt/chef-automate/backups` location unless you specify the location in the `config.toml` file.
 
     ```bash
     chef-automate backup create
+    ```
+
+    Run the below command command to create the `bootstrap.abb` bundle. This bundle captures any local credentials or secrets not persisted in the database.
+
+    ```bash
     chef-automate bootstrap bundle create bootstrap.abb
     ```
 
-    - The first command will create the backup to the `/var/opt/chef-automate/backup` location unless you specify the location in `config.toml` file.
-    - The second command will create the `bootstrap.abb`.
-    - Once the backup is completed, save the backup Id. For example: `20210622065515`.
+    Once the backup is completed, save the backup Id. For example: `20210622065515`.
+
+1. If you haven't specified the location in the `config.toml` file, go to the `/var/opt/chef-automate/backups` location and create **Bundle** using the following command:
+
+    ```bash
+    tar -cvf backup.tar.gz <backup_id>/ automatebackup-elasticsearch/ .tmp/
+    ```
+
+1. Transfer the `tar` bundle to one of the Chef Automate HA Frontend Nodes using the following command:
```

Comment:
We have to transfer the file to Automate node, not the fronted node. Infact we have to do the restoration in Bastion node.